test/rules/no-for-in/test.ts.lint

```diff
@@ -0,0 +1,23 @@
+// this should pass
+const columns = [{ data: [1] }];
+for (let i = 0; i < columns[0].data.length; i++) {
+	columns.map((x) => x.data[i]);
+}
+
+// this should NOT pass
+const object = {test:1, test2:1 ,test3:1};
```

Comment:
can you please format this code so that it would pass prettier? write it in a normal `.ts` file in the repo and use your editor to format it. right now there are a lot of whitespace inconsistencies

Reply:
```suggestion
const object = { test: 1, test2: 1, test3: 1 };
```